ecosystem/sep-0008.md

```diff
@@ -88,79 +83,62 @@ Parameters:
 Name | Data Type | Description
 -----|-----------|------------
 status|string|`"success"`
-tx|string|Transaction Envelope XDR, Base64 encoded. This transaction will have both the original signature(s) from the request, as well as an additional issuer signature.
+tx|string|Transaction envelope XDR, base64 encoded. This transaction will have the original signature(s) from the request as well as the asset issuer's signature.
 message|string|A human readable string containing information to pass on to the user (optional).
 
-#### Revised Response
-
-A Revised response will have a `200` HTTP status code and `revised` as the `status` value. It means that the transaction was revised to be made compliant. The user should examine and resign the transaction.
-
-Parameters:
-
-Name | Data Type | Description
------|-----------|------------
-status|string|`"revised"`
-tx|string|Transaction Envelope XDR, Base64 encoded. This transaction is a revised compliant version of the original request transation, signed by the issuer.
-message|string|A human readable string explaining the modifications made to the transaction to make it compliant.
-
-#### Pending Response
-
-A Pending response will have a `200` HTTP status code and `pending` as the `status` value. It means that the issuer needs to asynchronously validate the transaction. The user should resubmit the transaction after a given timeout.
-
-Parameters:
-
-Name | Data Type | Description
------|-----------|------------
-status|string|`"pending"`
-timeout|integer|Number of milliseconds to wait before submitting the same transaction again.
-message|string|A human readable string containing information to pass on to the user (optional).
-
-#### Action Required Response
-
-An Action Required response will have a `200` HTTP status code and `action_required` as the `status` value. It means that the user must complete an action before this transaction can be approved. The approval service will provide a URL that facilitates the action. Upon completion, the user will resubmit the transaction.
-
-Parameters:
-
-Name | Data Type | Description
------|-----------|------------
-status|string|`"action_required"`
-message|string|A human readable string containing information regarding the action required.
-action_url|string|A URL that allows the user to complete the actions required to have the transaction approved.
```

Comment:
❓ Are you sure an issuer would not need action required? This change appears to be focused on replacing the cosigning attribute of SEP-8's approach with the use of the allow trust sandwich, and removing the action required is independent of that.

Reply:
I'm not sure as I haven't been able to gather information about having which status will facilitate the issuers the most. I was simply thinking that this can be combined with the `rejected` status by putting the `action required` message in the error message.

Having said that, do you think changing the response would be a problem if this were to be done in a new SEP?

Reply:
If we're doing this in a new SEP then I agree, don't add anything you don't know of needing. Just the fact we had it here suggests someone thought it was needed so it might be worth asking around. If there's no evidence though, yeah I'd leave it out in a new SEP.

Reply:
Combine it with `rejected`: https://github.com/stellar/stellar-protocol/pull/750/commits/051a787a8c7cdb0204c7cf6683001e4bfba54e57